packages/keyring-controller/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [19.0.2]
+
+### Changed
+
+- Remove use of `@metamask/keyring-api` ([#4695](https://github.com/MetaMask/core/pull/4695))
+  - `@metamask/providers` and `webextension-polyfill` peer depedencies are no longer required.
+- Use new `@metamask/keyring-internal-api@^1.0.0` ([#4695](https://github.com/MetaMask/core/pull/4695))
+  - This package has been split out from the Keyring API. Its types are compatible with the `@metamask/keyring-api` package used previously.
+- Bump `@metamask/message-manager` from `^11.0.2` to `^11.0.3` ([#5048](https://github.com/MetaMask/core/pull/5048))
+- Bump `jsonschema` from `^1.2.4` to `^1.4.1` ([#4998](https://github.com/MetaMask/core/pull/4998)), ([#5027](https://github.com/MetaMask/core/pull/5027))
```

Comment:
It looks like the only change to `keyring-controller` was bumping (and then un-bumping) `eth-sig-util`, is that right?

``` suggestion
```

Reply:
The `eth-sig-util` has been updated and then reverted right away yes. However, the `jsonschema` has still been updated:
https://github.com/MetaMask/core/blame/main/packages/message-manager/package.json#L55

Reply:
That `jsonschema` package seems to apply to `message-manager` and not `keyring-controller`. Is that true?

Reply:
You're absolutely right 🙈 another good catch! I'll go with your suggestion then!